bftengine/src/bftengine/messages/CheckpointMsg.hpp

```diff
@@ -46,7 +52,7 @@ class CheckpointMsg : public MessageBase {
     uint8_t flags;
   };
 #pragma pack(pop)
-  static_assert(sizeof(CheckpointMsgHeader) == (2 + 8 + DIGEST_SIZE + 1), "CheckpointMsgHeader is 43B");
+  static_assert(sizeof(CheckpointMsgHeader) == (6 + 8 + DIGEST_SIZE + 1), "CheckpointMsgHeader is 49B");
```

Comment:
nit - you add 4 bytes, so it should be 47B, right?

Reply:
Done